lib/providers/restful-provider.js

```diff
@@ -0,0 +1 @@
+module.exports = function(){};
```

Comment:
probably worth noting that i didn't implement this yet; i just matched the featureset we had, and we didn't have a restful provider. this will need to be added in the coming days